api/types/license.go

```diff
@@ -123,6 +123,12 @@ type License interface {
 	GetAccountID() string
 
 	// GetFeatureSource returns where the features should be loaded from.
+	//
+	// Deprecated.
```

Comment:
Hmm, is this really deprecated? If we're still using it for the foreseeable future I dont think we should call it deprecated.

Who is going to update Cloud Enterprise to read features from Sales Center?

@bl-nero @capnspacehook are we good with making EUBP read from SC?

cc @bl-nero @capnspacehook @mcbattirola

Reply:
> Who is going to update Cloud Enterprise to read features from Sales Center?

this [PR](https://github.com/gravitational/teleport.e/pull/2708) starts having `cloud enterprise` read from sales center, with this cloud enterprise will start periodically pinging salescenter for features

this [PR](https://github.com/gravitational/teleport.e/pull/2763) handles reading for EUB/Team